zjit/src/cruby.rs

```diff
@@ -738,6 +738,11 @@ fn ruby_str_to_rust(v: VALUE) -> String {
     }
 }
 
+pub fn ruby_sym_to_rust(v: VALUE) -> String {
```

Comment:
```suggestion
pub fn ruby_sym_to_rust_string(v: VALUE) -> String {
```

It's not the Rust itself, so the name feels a bit weird. Given that we have `cstr_to_rust_string`, maybe we should call it `rust_string`? Same applies to `ruby_str_to_rust` above this.

If that's too long, I'd just call it `string` instead of `rust_string`.

Reply:
I went for matching function immediately above but I agree that both names need improvemen